plugins/adrv9002.c

```diff
@@ -1205,6 +1091,122 @@ static void load_profile(GtkFileChooser *chooser, gpointer data)
 		gtk_file_chooser_set_filename(chooser, "(None)");
 }
 
+// profile generator default config structures
+static adrv9002_config lte_defaults(void)
+{
+	tx_radio_channel_config tx1;
+	tx1.enabled = true;
+	tx1.sample_rate_hz = 61440000;
+	tx1.frequency_offset_correction_enable = false;
+	tx1.analog_filter_power_mode = 2;
+	tx1.channel_bandwidth_hz = 38000000;
+	tx1.elb_type = 0;
+	tx1.orx_enabled = false;
+
+	tx_radio_channel_config tx2 = tx1;
+
+	rx_radio_channel_config rx1;
+	rx1.enabled = true;
+	rx1.sample_rate_hz = 61440000;
+	rx1.frequency_offset_correction_enable = false;
+	rx1.analog_filter_power_mode = 2;
+	rx1.channel_bandwidth_hz = 38000000;
+	rx1.adc_high_performance_mode = true;
+	rx1.analog_filter_biquad = false;	    // got from default cfg
+	rx1.analog_filter_bandwidth_hz = 18000000; // got from default cfg
+	rx1.nco_enable = false;
+	rx1.nco_frequency_hz = 0;
+	rx1.rf_port = 0;
+
+	rx_radio_channel_config rx2 = rx1;
```

Comment:
Couple of notes from me (just about good practices :)):

1) I would have done this move before the previous patch... Thus you would not need to still have that plain 2 in the previous patch when removing CHANNEL_COUNT.
2) You should mention in your commit message that while reducing the scope you removed the for() loop by implicit memcpy(). I was actually wondering about the loops before realizing the subtle change. If you mention it, it's then clear what's going on.

On a different note (for future refactor)... Please consider note doing things like the above function where you're returning complete objects like `adrv9002_config` (which are likely not that small). That's inefficient and, honestly, far from great C. Use pointers for things like this (likely as an argument of the function).